behat.yml.dist

```diff
@@ -4,7 +4,7 @@ default:
       paths:
         features: '%paths.base%/tests-behat'
       contexts:
-        - atk4\ui\behat\ContextDump
+        - atk4\ui\behat\Context
```

Comment:
I am ok with this change as `ContextDump` is not much helpful with AJAX...

Reply:
ContextDump is useful when testing locally though.

Reply:
Yes, but static page issues are discovered even with phpunit... Marking as resolved.